### ISSUE TITLE
Support fully qualified paths to plugins

### ISSUE THREAD

**Issue by KylePDavis** [webdriverio/webdriverio#4867]
**Is your feature request related to a problem? Please describe.**
Currently [`initialisePlugin()`](https://github.com/webdriverio/webdriverio/blob/v5.16.15/packages/wdio-utils/src/initialisePlugin.js#L13) does direct module resolution only when the package name starts with `@`.

I want to be able to use fully resolved paths for plugins so that I can build a shared config package with shared dependencies and properly reference the bundled dependencies in a way that the consumer of this package can use them properly.

**Describe the solution you'd like**
I would like for this to also do direct module resolution when the package name starts with a slash.

So maybe we could change [`initialisePlugin()`](https://github.com/webdriverio/webdriverio/blob/v5.16.15/packages/wdio-utils/src/initialisePlugin.js#L13) like this:
```diff
- if (name[0] === '@') {
+ if (name[0] === '@' || name[0] === path.sep) {
```

**Describe alternatives you've considered**
Another approach might be to allow pre-loaded plugins in more situations (e.g., `services` can already take a function). This would unfortunately require loading these plugins during config definition rather than config usage. I suspect that we'd rather continue to lazily load plugins during config usage if possible.

**Additional context**
Here's a more concrete example of what I'd like to see work:
```js
{
  runner: require.resolve("@wdio/local-runner"),
  framework: require.resolve("@wdio/mocha-framework"),
  reporters: [
    require.resolve("@wdio/spec-reporter")
  ]
}
```

**Comment by christian-bromann** [comment]
> So maybe we could change [`initialisePlugin()`](https://github.com/webdriverio/webdriverio/blob/v5.16.15/packages/wdio-utils/src/initialisePlugin.js#L13) [](/webdriverio/webdriverio/blob/HEAD@{2019-12-11T01:59:46Z}/packages/wdio-utils/src/initialisePlugin.js#L13) like this:

Your suggested approach makes sense. Would you mind preparing a PR for it?

**Comment by christian-bromann** [comment]
Fixed in #4959
